specification/dimensions/pricingcategory.md

```diff
@@ -0,0 +1,47 @@
+# Pricing Category
+
+Pricing Category describes the kind of pricing model used for a charge at the time it was used or purchased. Pricing Category is commonly used for scenarios like differentiating between charges at on-demand vs. reduced prices when investigating optimization opportunities, like for commitment-based discount coverage.
```

Comment:
```suggestion
Pricing Category describes the pricing model used for a charge at the time of use or purchase. It can be useful for distinguishing between charges with [List Unit Prices] #ListUnitPrice or reduced prices, exposing optimization opportunities such as increasing commitment-based discount coverage
```